HPXMLtoOpenStudio/resources/defaults.rb

```diff
@@ -5591,6 +5821,317 @@ def self.get_ceiling_fan_months(weather)
+        watts += HVAC.get_blower_fan_power_watts(heat_pump.fan_watts_per_cfm, heat_pump.cooling_airflow_cfm)
+
+        if heat_pump.fraction_heat_load_served == 0
+          breaker_spaces += get_breaker_spaces_from_power_watts(HVAC.get_dx_cooling_coil_power_watts_from_capacity(UnitConversions.convert(heat_pump.cooling_capacity, 'btu/hr', 'kbtu/hr')), voltage) # ODU; the ~2 we missed adding to heating
+        end
+      end
+
+    elsif type == HPXML::ElectricPanelLoadTypeWaterHeater
+      hpxml_bldg.water_heating_systems.each do |water_heating_system|
+        next if !system_ids.include?(water_heating_system.id)
+        next if water_heating_system.fuel_type != HPXML::FuelTypeElectricity
+        next if water_heating_system.is_shared_system
+
+        if water_heating_system.water_heater_type == HPXML::WaterHeaterTypeStorage
+          watts += UnitConversions.convert(water_heating_system.heating_capacity, 'btu/hr', 'w')
+          breaker_spaces += 1
+        elsif water_heating_system.water_heater_type == HPXML::WaterHeaterTypeHeatPump
+          watts += [UnitConversions.convert(Waterheater.get_heating_input_capacity(water_heating_system.heating_capacity, water_heating_system.additional_properties.cop), 'btu/hr', 'w'),
+                    UnitConversions.convert(water_heating_system.backup_heating_capacity, 'btu/hr', 'w')].max
+          if voltage == 240
+            breaker_spaces += 1
+          end
+        elsif water_heating_system.water_heater_type == HPXML::WaterHeaterTypeTankless
+          if hpxml_bldg.building_construction.number_of_bathrooms == 1
+            watts += 18000
+          elsif hpxml_bldg.building_construction.number_of_bathrooms == 2
+            watts += 24000
+          else # 3+
+            watts += 36000
+          end
+          breaker_spaces += 1
+        end
+        breaker_spaces += 1
+      end
+
+    elsif type == HPXML::ElectricPanelLoadTypeClothesDryer
+      hpxml_bldg.clothes_dryers.each do |clothes_dryer|
+        next if !system_ids.include?(clothes_dryer.id)
+        next if clothes_dryer.fuel_type != HPXML::FuelTypeElectricity
+
+        if clothes_dryer.is_vented
+          watts += 5760 # FIXME: not 2640?
+          breaker_spaces += 1
+        else # HP
+          if voltage == 120
+            watts += 996
+          else
+            watts += 860
+            breaker_spaces += 1
+          end
+        end
+        breaker_spaces += 1
+      end
+    elsif type == HPXML::ElectricPanelLoadTypeDishwasher
+      hpxml_bldg.dishwashers.each do |dishwasher|
+        next if !system_ids.include?(dishwasher.id)
+
+        watts += 1200
+        breaker_spaces += 1
+      end
+    elsif type == HPXML::ElectricPanelLoadTypeRangeOven
+      hpxml_bldg.cooking_ranges.each do |cooking_range|
+        next if !system_ids.include?(cooking_range.id)
+        next if cooking_range.fuel_type != HPXML::FuelTypeElectricity
+
+        if voltage == 120
+          watts += 1800
```

Comment:
Is 120V being assigned to ranges based on the `IsInduction` boolean?

Reply:
No. I'm defaulting all electric ranges to 240V. If instead specified as 120V, 1800 W gets applied (induction or not). This seems to be what https://github.com/NREL/resstock/blob/electrical_panels/Electrical_Panels_EUSS/panels/nameplate_rating_new_load.csv implies, no?

Reply:
Yes, I think you're right. Just clarifying that 120V would have to be a direct argument, not automatically assigned for induction since I couldn't find it in the code